### PR TITLE
 added double axis and updated exomol molecules list and get_molecule method of DataFileManager class

### PR DESCRIPTION
**Pull request by 1someshverma**
This is to solve issuse https://github.com/radis/radis/issues/446 on Radis repository
Also, I have updated Exomol Molecules list and added ZrO, Na and K in it the list.
Plot with double axis (nm, cm-1) https://github.com/radis/radis/issues/446

It provides the secondary or double x axis at the top of plot
-When the plot is between wavenumber(cm-1 )vs radiance , it adds a secondary axis for wavelength(nm)
-When the plot is between radiance vs wavelenght , it also adds a secondary axis for wavenumber(cm-1)

Fixes https://github.com/radis/radis/issues/446

 
Screenshot from 2023-01-16 01-09-20

Edit : Updated code to use functions defined in .pysics.convert and now wavelength is obtained depending upon whether medium is air or vaccum

### CODE REVIEW

File: radis/spectrum/spectrum.py [hunk 1]
```diff
@@ -2137,26 +2140,46 @@ def clean_error_msg(string):
...
         fix_style()
 
+        def waveLength2waveNumber(x):
+            return 1e4/x
+
+        def waveNumber2waveLength(x):
+            return 1e4/x
+
+        print(xlabel)
+        print(ylabel)
```
Comment (erwanp) [inline radis/spectrum/spectrum.py:2174]: this can be removed

File: radis/spectrum/spectrum.py [hunk 2]
```diff
@@ -2137,26 +2140,46 @@ def clean_error_msg(string):
...
+        ax.set_xlabel(xlabel)
+        ax.set_ylabel(ylabel)
+        ax.set_yscale(yscale)
         if "label" in kwargs:
-            plt.legend()
+            ax.legend()
         fix_style()
 
+        def waveLength2waveNumber(x):
+            return 1e4/x
```
Comment (erwanp) [inline radis/spectrum/spectrum.py:2168]: There are cm2nm and nm2cm functions in RADIS (.phys.convert maybe?), can you use them? 

Btw you might find a problem : wavelength in vacuum are slightly different from wavelength in air. Assume wavelength in air if wunit is "nm" or "nm_air", assume wavelength in vacuum if "nm_vac"

**Review by erwanp — CHANGES_REQUESTED** [review]

File: radis/spectrum/spectrum.py (hunk 1 above)
Comment (1someshverma) [reply radis/spectrum/spectrum.py:2174]: ok, i will remove it. I had added these to check certain things but now i'll remove these

File: radis/spectrum/spectrum.py (hunk 2 above)
Comment (1someshverma) [reply radis/spectrum/spectrum.py:2168]: sure, i'll update the code using the functions defined in the .physics.convert and also take care to of wavelength in air and vaccum

**Comment by 1someshverma** [comment]
i have done the changes

**Review by erwanp — APPROVED** [review]
Very good, let's merge

**Comment by erwanp** [comment]
Hello Somesh, we have problems when plotting multiple graphs on the same figure, the secondary axis gets duplicated and it doesn't render well. Example : 

```

    from radis.lbl import SpectrumFactory

    setup_test_line_databases()  # add HITRAN-CO-TEST in ~/radis.json if not there

    Tgas = 1500
    sf = SpectrumFactory(
        wavelength_min=4400,
        wavelength_max=4800,
        mole_fraction=0.01,
        #                         path_length=0.1,
        cutoff=1e-25,
        wstep=0.005,
        isotope=[1],
        self_absorption=True,
        verbose=verbose,
    )
    sf.warnings["MissingSelfBroadeningWarning"] = "ignore"
    #        sf.warnings['NegativeEnergiesWarning'] = 'ignore'
    sf.load_databank("HITRAN-CO-TEST")
    s1 = sf.non_eq_spectrum(Tgas, Tgas, path_length=0.01)
    s2 = sf.non_eq_spectrum(Tgas, Tgas, path_length=3)
    s1.rescale_path_length(3)

    if plot:
        fig = plt.figure(fig_prefix + "Rescaling path length")
        s2.plot("radiance_noslit", nfig=fig.number, lw=3, label="L=3m")
        s1.plot(
            "radiance_noslit",
            nfig=fig.number,
            color="r",
            label="L=0.01m, rescaled to 3m",
        )
        plt.title("Non optically thin rescaling")
        plt.legend()
        plt.tight_layout()

```

image


Can you do something about it ?

**Pull request merged**